vite.config.js

```diff
@@ -0,0 +1,6 @@
+import glsl from "vite-plugin-glsl";
+import { defineConfig } from "vite";
+
+export default defineConfig({
+  plugins: [glsl.default()] //For some reason, cannot resolve glsl(). Workaround found on vite-plugin-glsl issue (https://github.com/UstymUkhman/vite-plugin-glsl/issues/1#issuecomment-972660771)
```

Comment:
Thanks for leaving a comment about this! 🙌

Reply:
I think the reason for this might be [this](https://github.com/mayacoda/simple-threejs-typescript-starter/blob/main/package.json#L15) line. It forces all JS files to be loaded as ES6 Modules, but `video-glsl-plugin` compiles to [`CommonJS`](https://github.com/UstymUkhman/vite-plugin-glsl/blob/main/tsconfig.json#L17), that's why you need to use it as `glsl.default()`. Since this seems to be a front-end project, I don't think you really need that line in `package.json`. More info [here](https://nodejs.org/api/packages.html#modules-packages).